src/hooks/usePublishCoordinator.js

```diff
@@ -0,0 +1,89 @@
+import partititon from 'lodash/partition';
+import {
+  useCallback,
+  useRef
+} from 'react';
+
+import {
+  useOkapiKy,
+  useStripes,
+} from '@folio/stripes/core';
+
+export const PUBLISH_COORDINATOR_STATUSES = {
+  COMPLETE: 'COMPLETE',
+  ERROR: 'ERROR',
+  IN_PROGRESS: 'IN_PROGRESS',
+};
+
+export const PUBLICATIONS_API = 'publications';
+export const CONSORTIA_API = 'consortia';
+
+
+export const TIMEOUT = 2500;
+
+const formatPublicationResult = ({ publicationResults, totalRecords }) => {
+  const [results, errors] = partititon(publicationResults, ({ statusCode }) => statusCode >= 200 && statusCode < 300);
```

Comment:
I suggest to move this callback in checker function like isValied response or smth lite that

Reply:
that logic was taken from ui-consortia-settings.https://github.com/folio-org/ui-consortia-settings/blob/master/src/hooks/usePublishCoordinator/usePublishCoordinator.js. We just decided to copy that function coz we don't want to add dependency consortia in bulk